packages/token/src/erc20/extensions/erc4626/erc4626.cairo

```diff
@@ -138,32 +146,43 @@ pub mod ERC4626Component {
     ///
     /// NOTE: To transfer fees, this trait needs to be coordinated with
     /// `ERC4626Component::ERC4626Hooks`.
-    ///
-    /// See the ERC4626FeesMock example:
+    /// See the ERC4626AssetsFeesMock and ERC4626SharesFeesMock examples:
     /// https://github.com/OpenZeppelin/cairo-contracts/tree/main/packages/test_common/src/mocks/erc4626.cairo
     pub trait FeeConfigTrait<TContractState, +HasComponent<TContractState>> {
-        /// Adjusts deposits within `preview_deposit` to account for entry fees.
-        /// Entry fees should be transferred in the `after_deposit` hook.
-        fn adjust_deposit(self: @ComponentState<TContractState>, assets: u256) -> u256 {
-            assets
+        /// Calculates the entry fee for a deposit during `preview_deposit`. The returned `Fee` is
+        /// advisory only. It must be handled (e.g., transferred) manually in the `after_deposit`
+        /// hook.
```

Comment:
We need to update the wording in this comments, since the fee is not advisory only from the fact that the preview functions and the actual execution is affected by the fee, even if this one is not transferred. This should be clear from doc comments.

Applies fot the other functions.